components/aws/sagemaker/batch_transform/component.yaml

```diff
@@ -71,7 +71,7 @@ outputs:
   - {name: output_location,    description: 'S3 URI of the transform job results.'}
 implementation:
   container:
-    image: carowang/kubeflow-pipeline-aws-sm:20190728-01
+    image: carowang/kubeflow-pipeline-aws-sm:20190801-01
```

Comment:
Can we move this image is to somewhere in AWS official dockerhub ?

Reply:
Yes will do once I get more info on how to do so